packages/withRequired/jest.config.ts

```diff
@@ -0,0 +1,9 @@
+import baseConfig from "../../jest.config.base";
```

Comment:
We should probably just not have this file if no tests

Reply:
I added tests instead